src/components/timeline/FlameChartCanvas.js

```diff
@@ -227,23 +218,23 @@ class FlameChartCanvas extends PureComponent {
       // Only JavaScript functions have a filename.
       const fileNameIndex = thread.funcTable.fileName[funcIndex];
       if (fileNameIndex !== null) {
-        resourceOrFileName = (
-          <div className="tooltipOneLine tooltipDetails">
-            <div className="tooltipLabel">File:</div>
-            {thread.stringTable.getString(fileNameIndex)}
-          </div>
-        );
+        resourceOrFileName = [
+          <div className="tooltipLabel" key="file">
+            File:
+          </div>,
+          thread.stringTable.getString(fileNameIndex),
+        ];
```

Comment:
for grid layout to work, we need all elements to be direct child of the grid parent. Hence the use of arrays here so that the content can be easily appended later.